l10n_it_delivery_note/readme/USAGE.rst

```diff
@@ -53,7 +53,7 @@ Di seguito, una piccola lista di passi da seguire per portare a termine la migra
 
 4. Lanciare Odoo, nella maniera in cui si è soliti fare, aggiungendo alcuni parametri al comando d’avvio:
 
-   `./odoo-bin migrate_ddt_data --database <nome_database> [...]`
+   `./odoo-bin migratel10nitddt --database <nome_database> [...]`
```

Comment:
Mi chiedo se, suddetto comando, debba essere ancora integrato anche nella versione 14.0 del modulo... 🤔

Reply:
Teoricamente no ma ad esempio mi è capitato ancora di doverlo usare in una migrazione da `10.0` a `14.0`.

Reply:
Oh, ok... Chiarissimo, allora! 😉